OlinkAnalyze/vignettes/Vignett.Rmd

```diff
@@ -628,10 +628,9 @@ A list of objects of class *ggplot* (silently returned). Plots are also printed
 
 ## Boxplots for outcomes (olink_boxplot)
 
-The olink_boxplot function is used to generate boxplots of NPX values stratified on a variable for a given list of proteins. olink_boxplot uses the functions *ggplot* and *geom_boxplot* of the R library *ggplot2*.
-
+The olink_boxplot function is used to generate boxplots of NPX values stratified on a variable for a given list of proteins.
 
```

Comment:
remove empty line and make these 2 sentences in one paragraph

Reply:
Merged into a single paragraph